modules/ROOT/pages/gov-cloud-load-balancer.adoc

```diff
@@ -0,0 +1,100 @@
+also use mapping rules to rename requests to the Government Cloud dedicated load balancer to a different Mule 
+application domain name.
+
+The Government Cloud dedicated load balancer exposes an external domain name that resolves to two public IP 
+addresses which are accessible from outside your Government Cloud Anypoint VPC network. `_lb-name_` is the name you 
+gave the load balancer when you created it:
+
+----
+ _lb-name.lb-gprod-rt.anypointdns.net
+----
+
+== Renaming External Mule Application URLs Through a Dedicated Load Balancer
+
+You can create a mapping rule to rename the Mule application, so it can be accessed through a different domain name.
+
+For example, suppose you deploy a Mule application named myApp-myCompany-prod.cloudhub.io. Government Cloud domain names 
+must be globally unique among every other MuleSoft Government Cloud customer and Mule application. A dedicated load balancer helps 
+you hide this naming complexity within your corporate DNS domain name.
+
+You can set a mapping rule in your dedicated load balancer so external clients can access the Mule application 
+on `+http://myApp.lb-name.lb-gprod-rt.anypointdns.net+`, or `+https://myApp.lb-name.lb-gprod-rt.anypointdns.net:443+`.
+
+In addition, you can set a CNAME record in your corporate DNS nameserver to mask the `lb-name.lb-gprod-rt.anypointdns.net` 
+domain to your company's "vanity" domain.
+
+For example, suppose your company owns the DNS domain `example.com`. You can create a CNAME record to route requests 
+to `myapp.example.com` to `myApp.lb-name.lb-gprod-rt.anypointdns.net`. This hides the complexity of the Government Cloud domain name 
+in the `cloudhub.io` domain.
+
+== Routing Internal Requests to the Dedicated Load Balancer
+
+The Government Cloud dedicated load balancer has an internal domain name that is used by applications and clients 
+within the Anypoint VPC.
+
+The internal domain uses the following naming convention (`_lb-name_` is  the name you gave the load balancer 
+when you created it):
+
+----
+internal-_lb-name_.lb-gprod-rt.anypointdns.net
+----
+
+
+Each Dedicated Load Balancer has a DNS A record `_lb-name_.lb-gprod-rt.anypointdns.net` that resolves to the two public 
+IP addresses of the two instances. Through your DNS provider, you can add a CNAME record pointing to this A record 
+and use your domain names to access it.
+
+If you want your load balancer to manage all connections to your application but do not want your default domain 
+name for your application publicly exposed, then each application must listen on HTTP port 8091 or 8092. Alternatively, 
+you can create a custom mapping policy to redirect outside requests from your load balancer to your specific application.
+
+The following graphic shows how a load balancer interacts between a VPC and Anypoint Platform.
+
+image::pxty-lb.png[Load Balancer]
+
+Your load balancer listens for outside requests over HTTPS and, by default, communicates internally with your worker 
```

Comment:
It is still unclear to me, representing the new user, how 
load balancer + outside requests + HTTPS + worker + HTTP 
relates to 
Mule app + VPC + HTTPS + upstream protocol + mapping